assets/protos/member.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+option csharp_namespace = "Skyra.Grpc.Services";
+
+package member;
+
+// The greeting service definition.
+service Member {
+  // Sends a greeting
+  rpc AddPoints (Points) returns (Result);
+  rpc GetPoints (MemberQuery) returns (Result);
+}
+
+// The request message containing the user's name.
+message Points {
```

Comment:
I think we should have some naming conventions for gRPC types - I am personally in favor of [Type]{Request | Result} - in this case, it'd be `PointsRequest` and the endpoint would return a `PointsResult`

Reply:
We're using `Query` just below, should I rename it to `Request`?

Reply:
imo, we're going to be adding a load of other services later down the line - and if we have each of them have a message named `Query` and also a message named `Request` - we're going to have about 20 different classes named `Query` and `Request` - which is going to reallllly bloat intellisense, so prefixing them with a sensible name is better.